src/CAN/FakeCANBoard.cpp

```diff
@@ -101,6 +109,48 @@ int main() {
 
 			int angle_target = prompt("Enter PID target (in 1000ths of degrees)");
 			can::motor::setMotorPIDTarget(serial, angle_target);
+		} else if (testMode == TestMode::PIDVel) {
+			if (!mode_has_been_set) {
+				// set pid mode
+				can::motor::setMotorMode(serial, motormode_t::pid);
+				mode_has_been_set = true;
+
+				// get pid coeffs
+				int p_coeff = prompt("P");
+				int i_coeff = prompt("I");
+				int d_coeff = prompt("D");
+
+				// set pid coeffs
+				can::motor::setMotorPIDConstants(serial, p_coeff, i_coeff, d_coeff);
+
+				// create can motor
+				double posScale =
+					prompt("Enter the positive scale for the motor (double value)\n");
+				double negScale =
+					prompt("Enter the negative scale for the motor (double value)\n");
+				motor = robot::can_motor(motorid_t::frontLeftWheel, true, serial, posScale,
+										 negScale);
+
+				// create velocity command
+				targetVel = prompt("Enter the target velocity (in millidegrees per second)\n");
+				motor.setMotorVel(targetVel);
+				startTime = robot::types::dataclock::now();
+			}
+
+			// get x data: current time
+			robot::types::datatime_t currTime = robot::types::dataclock::now();
+
+			// get y data: set point (target vel * time since set velocity call)
+			double setPoint = targetVel * fabs((currTime.time_since_epoch().count() -
+												startTime.time_since_epoch().count()));
```

Comment:
I believe this will multiply by the elapsed time in millis, but we need seconds. We can be pretty explicit about it though by first doing `util::durationToSec(currTime - startTime)`, which will give us elapsed time in seconds.

Reply:
Also add the initial position to the setpoint, it's basically the +C since we're integrating velocity.